packages/components/scrollbar/src/scrollbar.ts

```diff
@@ -10,10 +11,7 @@ export const scrollbarProps = buildProps({
     type: [String, Number],
     default: '',
   },
-  native: {
-    type: Boolean,
-    default: false,
-  },
+  native: Boolean,
```

Comment:
Why did you change it? I think we can keep it as-is.

Reply:
I think the default value of Boolean props is `false`, so I simplified the code. 
refer to: https://github.com/vuejs/core/issues/5863#issuecomment-1118446966